specs/altair/validator.md

```diff
@@ -49,8 +49,8 @@ This is an accompanying document to [Ethereum 2.0 Altair -- The Beacon Chain](./
 
 ## Introduction
 
-This document represents the expected behavior of an "honest validator" with respect to the Altair upgrade of the Ethereum 2.0 protocol.
-It builds on the [previous document for the behavior of an "honest validator" from Phase 0](../phase0/validator.md) of the Ethereum 2.0 protocol.
+This document represents the expected behavior of an "honest validator" with respect to the Altair upgrade of the Ethereum consensus protocol.
+It builds on the [previous document for the behavior of an "honest validator" from Phase 0](../phase0/validator.md) of the Ethereum consensus protocol.
```

Comment:
Ah, I just saw you used `Ethereum proof-of-stake protocol` to replaced them on phase0 spec. Either way is fine with me.

```suggestion
This document represents the expected behavior of an "honest validator" with respect to the Altair upgrade of the Ethereum proof-of-stake protocol.
It builds on the [previous document for the behavior of an "honest validator" from Phase 0](../phase0/validator.md) of the Ethereum proof-of-stake protocol.
```

Reply:
modded back to pos for consistency